src/components/NavigationSection.svelte

```diff
@@ -69,6 +76,22 @@
     }
   }
 
+  function getCategories (items) {
+    const categoriesOject = {}
+    items.forEach(i => {
+      if (!categoriesOject.hasOwnProperty(i.category)){
+        categoriesOject[i.category] = []
+      }
+      categoriesOject[i.category].push(i)
+    })
+    // since sveltejs doesn't allow to loop through objects
+    const categories = []
+    for (const category in categoriesOject) {
+      categories.push({title: category, items: categoriesOject[category]})
```

Comment:
Good point! I will rename this function to be more explicit. And @gregorylegarec it's `items` instead of `apps` because it's not specific to the apps here, you can use the [`categorized` property](https://github.com/cozy/cozy-bar/pull/51/files#diff-19e2638c4d86ae90b81f1d3f8354850dR36) to categorize other items in general.